build.gradle

```diff
@@ -17,48 +17,18 @@ android {
 
     buildTypes {
         debug {
-            buildConfigField "String", "CLIENT_ID", "\"" + getGHId(false) + "\""
-            buildConfigField "String", "CLIENT_SECRET", "\"" + getGHSecret(false) + "\""
-            buildConfigField "String", "CLIENT_CALLBACK", "\"" + getGHCallback(false) + "\""
         }
         release {
-            buildConfigField "String", "CLIENT_ID", "\"" + getGHId(true) + "\""
-            buildConfigField "String", "CLIENT_SECRET", "\"" + getGHSecret(true) + "\""
-            buildConfigField "String", "CLIENT_CALLBACK", "\"" + getGHCallback(true) + "\""
         }
     }
 }
 
-def getGHId(pro) {
-    if (pro) {
-        return hasProperty('GH_PRO_ID') ? GH_PRO_ID : System.getenv('GH_PRO_ID')
-    } else {
-        return hasProperty('GH_DEV_ID') ? GH_DEV_ID : System.getenv('GH_DEV_ID')
-    }
-}
-
-def getGHSecret(pro) {
-    if (pro) {
-        return hasProperty('GH_PRO_SECRET') ? GH_PRO_SECRET : System.getenv('GH_PRO_SECRET')
-    } else {
-        return hasProperty('GH_DEV_SECRET') ? GH_DEV_SECRET : System.getenv('GH_DEV_SECRET')
-    }
-}
-
-def getGHCallback(pro) {
-    if (pro) {
-        return hasProperty('GH_PRO_CALLBACK') ? GH_PRO_CALLBACK : System.getenv('GH_PRO_CALLBACK')
-    } else {
-        return hasProperty('GH_DEV_CALLBACK') ? GH_DEV_CALLBACK : System.getenv('GH_DEV_CALLBACK')
-    }
-}
-
 dependencies {
 
-    compile 'com.android.support:support-annotations:21.0.3'
+    compile 'com.android.support:support-annotations:22.2.0'
 
     compile 'net.danlew:android.joda:2.6.0'
     compile 'org.ocpsoft.prettytime:prettytime:3.2.4.Final'
 
-    compile project(':sdks:BaseSDK')
+    compile 'com.github.Meisolsson:BaseSDK:a81d3e4934'
```

Comment:
It's a thing I was testing. JitPack (https://jitpack.io/). It can be removed and replaced with the old system or it can be used (We were discussing it over at https://github.com/forkhubs/android/issues/792#issuecomment-113801212). I just forgot it was there.